osu.Game/Overlays/Direct/DirectPanel.cs

```diff
@@ -87,5 +260,30 @@ public Statistic(FontAwesome icon, int value = 0)
                 Value = value;
             }
         }
+
+        private class TransformEdgeEffectRadius : Transform<float, Container>

```

Comment:
We should have transform helpers for pretty much every property the framework exposes at least, imho.